01-data-structures/04-hashes-part-1/hashclass.rb

```diff
@@ -1,27 +1,59 @@
 class HashClass
+  require_relative 'hash_item'
 
   def initialize(size)
     @items = Array.new(size)
+    @size = size
```

Comment:
You do not have to fix this, but since you have and maintain this value, your calls to `@item.length` throughout the code are unnecessary.